addons/block-palette-icons/userstyle.css

```
@@ -0,0 +1,47 @@
+.scratchCategoryItemBubble {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 17px 17px;
+  height: 22px;
+  width: 22px;
+}
+
+.scratchCategoryId-motion .scratchCategoryItemBubble {
+  background-image: url(chrome-extension://npijjahcnpemcijdgoioaclneonckman/addons/block-palette-icons/icons/motion_icon.svg);
+}
+
+.scratchCategoryId-looks .scratchCategoryItemBubble {
+  background-image: url(chrome-extension://npijjahcnpemcijdgoioaclneonckman/addons/block-palette-icons/icons/looks_icon.svg);
+}
+
+.scratchCategoryId-sound .scratchCategoryItemBubble {
+  background-image: url(chrome-extension://npijjahcnpemcijdgoioaclneonckman/addons/block-palette-icons/icons/sound_icon.svg);
```

url() is relative. please use that, this probably won't work on anyone else's computer.
```suggestion
  background-image: url(sound_icon.svg);
```

---

oh yeah i was going to but forgot

p.s. actually it's a theme so it's injected into the page. but I can use `%addon-self-dir%`